examples/exotica_examples/scripts/example_ik_eff_axis_alignment.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+import rospy
+import pyexotica as exo
+import numpy as np
+from numpy import array
+from numpy import matrix
+import math
+from pyexotica.publish_trajectory import *
```

Comment:
Maybe just ``publishPose`` and ``sigIntHandler`` - we currently have hidden dependencies that are getting into examples by using the ``*``. Once we fix the ``publish_trajectory`` those examples would be broken.

Reply:
This is fine but what about `signal`? Should this be imported as an additional module or from `publish_trajectory`? I.e. 1 or 2 as follows.

1:

```
from pyexotica.publish_trajectory import publishPose, sigIntHandler, signal
```

2:

```
import signal
from pyexotica.publish_trajectory import publishPose, sigIntHandler
```

Reply:
(2) is the better option.